roles/openshift_manage_node/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+- name: Handle unscheduleable node
+  delegate_to: "{{ openshift_first_master }}"
```

Comment:
@detiber, I'm guessing this delegate_to should be removed too: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_node_certificates/tasks/main.yml#L24